### PR TITLE
Minor Updates in Readme.md

### PR DESCRIPTION
**Pull request by Jathurshan-2019**
Added How to install this file without Trusted Certificate
Fixed some Markdown errors in the original file
Made some elements look better

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
-Here are up to date renders and screenshots with all our current features:
+Created with WinUI and the latest design ideologies, Rise Media Player is modern while keeping all of the classic features people need. We use WinUI 2.6/2.7 to keep our user interface, clean, modern and consistent with Windows 11 UI. Although, this app does work on Windows 10 too. We use all of your favourite materials: Mica and Acrylic. We use Microsoft's icons and controls as well as some of our own to create a generally native feeling experience. Our own controls and icons give users a truly personalised experience, being able to choose their own icon packs and with features like compact mode, you can use it on any Windows device! *1 (go to bottom)
```
Comment (itsWindows11) [inline readme.md:24]:
```
with Windows 11 UI
```
Both UI and UX apply to this app, so you could include both.

File: readme.md [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+
+-----------------------------
+### **PRE ALPHA 2 OUT NOW! Download the installer from the download link below or clone the repository to build :)**
+---------------------------
 
 **Rise Media Player** brings all of your media to a whole new level. If it's videos, the music stored on your devices, discs or even your favourite streaming services; you'll love our player. One library for everything. Almost infinite customisability with settings that are second to none. Stream, browse and explore with Streaming Services - coming soon.
 
 **Note**: Videos are not available yet. This is a high priority for us and they will be coming very soon.
 
+Contribute to this Project by Github in [RMP](https://github.com/Rise-Software/Rise-Media-Player)
```
Comment (itsWindows11) [inline readme.md:21]:
```
Contribute to this Project by Github in [RMP](https://github.com/Rise-Software/Rise-Media-Player)
```
You could perhaps change this to the snippet below:

```
[Contribute to this Project](https://github.com/Rise-Software/Rise-Media-Player)
```

File: readme.md (hunk 1 above)
Comment (itsWindows11) [inline readme.md:24]:
```
with Windows 11 UI
```
Both UI and UX apply to this app, so you could include both.

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
Overall, this looks good, but there are changes I have suggested to you, you can read them above this comment.

I don't know how to properly review a pull request so sorry for the messed up review.

File: readme.md [hunk 3]
```diff
@@ -1,19 +1,32 @@
+# Rise Media player
```
Comment (itsWindows11) [inline readme.md:1]:
```suggestion
# Rise Media Player
```

**Comment by Jathurshan-2019** [comment]
That's ok, it is not a problem for me, I will correct it. Thank you for your correction

File: readme.md (hunk 1 above)
Comment (Jathurshan-2019) [reply readme.md:24]: Ok

**Comment by Jathurshan-2019** [comment]
Check whether it is ok or not

**Comment by itsWindows11** [comment]
> Check whether it is ok or not

I'll check it out again and get back to you when I'm done

File: readme.md [hunk 4]
```diff
@@ -68,23 +81,32 @@ Currently, all users can try out the MSIX packages. When we move to the store, y
...
+*1 Any Windows device running Windows 10 1903 or later. Windows Mobile/Windows Phone is not supported.*
+* If Windows Prompts about Untrusted Application
+* Simply right click and Go to Digital signatures Tab
+* Select Rise Software and click Details
+* In next popup select View Certificate
+* In next popup select Install Certificate
+* In next step select Local machine (If a user account prompt appear click yes)
+* Select Place all certificates in Following store and click browse select Trusted People and click OK and Next
+* Click Finish
+* Confirmation windows will appear After that Try Installing the MSIXBUNDLE
```
Comment (itsWindows11) [inline readme.md:112]:
```suggestion
* Confirmation windows will appear, after that try installing the MSIXBUNDLE
```

File: readme.md [hunk 5]
```diff
@@ -68,23 +81,32 @@ Currently, all users can try out the MSIX packages. When we move to the store, y
...
+
+### Notes
+*1 Any Windows device running Windows 10 1903 or later. Windows Mobile/Windows Phone is not supported.*
+* If Windows Prompts about Untrusted Application
+* Simply right click and Go to Digital signatures Tab
+* Select Rise Software and click Details
+* In next popup select View Certificate
+* In next popup select Install Certificate
+* In next step select Local machine (If a user account prompt appear click yes)
+* Select Place all certificates in Following store and click browse select Trusted People and click OK and Next
```
Comment (itsWindows11) [inline readme.md:110]:
```suggestion
* Select "Place all certificates in following store" and click browse, then select "Trusted People" and click OK and Next
```

File: readme.md [hunk 6]
```diff
@@ -68,23 +81,32 @@ Currently, all users can try out the MSIX packages. When we move to the store, y
...
-
+* [**Agnihotra Nath (@AlfarexGuy2019)**](https://github.com/alfarexguy2019)
+* [**Makesh Vineeth (@MakeshVineeth)**](https://github.com/makeshvineeth)
+* [**Rareș Pița (@dlphin)**](https://github.com/dlphin)
+------------------
+
+### Notes
+*1 Any Windows device running Windows 10 1903 or later. Windows Mobile/Windows Phone is not supported.*
+* If Windows Prompts about Untrusted Application
+* Simply right click and Go to Digital signatures Tab
```
Comment (itsWindows11) [inline readme.md:105]:
```suggestion
* If Windows prompts about Rise Media Player being an untrusted application, simply right click and go to "Digital signatures" tab
```

File: readme.md [hunk 7]
```diff
@@ -1,19 +1,31 @@
...
   <img src="https://user-images.githubusercontent.com/74561130/139312648-b60842df-9dda-4e31-ba29-649e7571d705.png" />
 </p>
 
-**PRE ALPHA 2 OUT NOW! Download the installer from the download link below or clone the repository to build :)**
+
+<p align="center">
+  Rise Media Player
+</p>
+
+
```
Comment (itsWindows11) [inline readme.md:12]: You don't need to put the app name three times here.

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
There are some other things you have to change here, too!

**Pull request merged**